crates/re_sdk/src/file_writer.rs

```diff
@@ -25,7 +25,7 @@ impl FileWriter {
 
         re_log::debug!("Saving file to {path:?}…");
 
-        let file = std::fs::File::create(&path).with_context(|| format!("Path: {:?}", path))?;
+        let file = std::fs::File::create(&path).with_context(|| format!("Path: {path:?}"))?;
```

Comment:
oh boy... here we go :smile:

Reply:
for some reason clippy started complaining here suddenly

Reply:
Well yes it's the famous "oh god I don't want to fix this one otherwise I'll have to justify why it passes the CI" lol #1447